tome/services/discord_client.py

```
@@ -0,0 +1,21 @@
+    try:
+        r = requests.post(
+            settings.BOT_ANNOUNCE_URL,
+            json=payload,
+            headers={"X-Api-Key": settings.BOT_TOKEN},
+            timeout=4,
+        )
+
+        if r.status_code >= 400:
+            print("Bot announce failed: %s %s", r.status_code, r.text[:500])
+
+    except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)